script/build-module.ts

```diff
@@ -11,28 +11,30 @@ import { parseBlogFile, parseFile, parseNav } from '../lib/parsers'
 import { IParseFile } from '../lib/interfaces'
 import locales, { IResult as ILocalesResult } from '../lib/locales'
 import { writeIndexFiles } from '../lib/generate-js'
+import { supportedVersions } from '../package.json'
 import {
   parseElectronGlossary,
   IParseElectronGlossaryReturn,
 } from '../lib/parse-electron-glossary'
 const getIds = require('get-crowdin-file-ids')
 
-const contentDir = path.join(__dirname, '../content/current')
+const contentDir = (version: string) =>
```

Comment:
nit: slightly better function name
```suggestion
const getContentDir = (version: string) =>
```